apps/DisplayCluster/resources/MasterContentWindow.qml

```diff
@@ -160,40 +288,4 @@ BaseContentWindow {
         anchors.topMargin: 10
         anchors.leftMargin: 10
     }
-
-    ContentWindowButton {
```

Comment:
Like that, windows border have priority over the buttons. Is it the wanted behaviour?